packages/ai/util/consume-stream.ts

```diff
@@ -9,9 +9,29 @@
  * @returns {Promise<void>} A promise that resolves when the stream is fully consumed.
  */
 export async function consumeStream(stream: ReadableStream): Promise<void> {
```

Comment:
Can you add tests for `consume-stream` that show the issue?

Reply:
I reverted some of my changes here to keep this low-level! Error handling (e.g. checking for abort errors) should be the responsbility of the caller